Swiftris/Swiftris/AppDelegate.swift

```diff
@@ -16,31 +16,40 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject: AnyObject]?) -> Bool {
         // Override point for customization after application launch.
+        NSLog("From AppDelegate: did finish launching")
         return true
     }
 
     func applicationWillResignActive(application: UIApplication) {
         // Sent when the application is about to move from active to inactive state. This can occur for certain types of temporary interruptions (such as an incoming phone call or SMS message) or when the user quits the application and it begins the transition to the background state.
         // Use this method to pause ongoing tasks, disable timers, and throttle down OpenGL ES frame rates. Games should use this method to pause the game.
-    }
+        NSLog("From AppDelegate: did resign active")
+
+        NSNotificationCenter.defaultCenter().postNotificationName("swiftrisDidEnterBackground", object: self)
```

Comment:
Posting a custom notification from `applicationWillResignActive()` is not necessary. The app will post a `UIApplicationWillResignActiveNotification` notification after the method finishes. It is this notification that you are listening for in GameViewController.